pkg/orchestration/wiring/edge/types.go

```diff
@@ -0,0 +1,46 @@
+package edge
+
+import "github.com/atlassian/voyager"
+
+type InstanceParameters struct {
+	ServiceName voyager.ServiceName `json:"serviceName"`
+	Resource    ResourceParameters  `json:"resource"`
+}
+
+type ResourceParameters struct {
+	Attributes Attributes `json:"attributes"`
+}
+
+type Attributes struct {
+	UpstreamAddress []UpstreamAddress `json:"upstream_address"`
+	UpstreamPort    int32             `json:"upstream_port,omitempty"`
+	UpstreamSuffix  string            `json:"upstream_suffix,omitempty"`
+	UpstreamOnly    string            `json:"upstream_only,omitempty"`
+	Domain          []string          `json:"domain,omitempty"`
```

Comment:
singular name for plural thing?

Reply:
this is an existing OSB contract (https://stash.atlassian.com/projects/MICROS/repos/resource-provisioning-service/browse/src/main/resources/globaledge_osb.json), so the format of OSB parameters don't bother me, as long as @cetanu got them right.

Reply:
This field was originally for a single domain, was changed later to allow multiple, bad name